basic_counter/lib/basic_counter_web/live/reaction_counter.ex

```diff
@@ -0,0 +1,77 @@
+defmodule CountersWeb.Live.ReactionCounter do
+  use CountersWeb, :live_component
+  @topic "reactions"
+
+  def render(assigns) do
+    ~H"""
+    <div class="my-4">
+      <h3 class="text-lg">
+        <span class="text-2xl"><%= @emoji %></span><%= render_slot(@inner_block) %>
+      </h3>
+      <div>
+        <button
+          class="bg-accent p-2 w-20 text-accent-foreground font-bold rounded-lg"
+          phx-click="inc"
+          phx-target={@myself}
+        >
+          🤩 <%= @count %>
+        </button>
+        Write took <%= @write_time |> us_to_ms %> ms, avg:
+        <%= @write_time_history |> get_avg_write_time |> us_to_ms %> ms
```

Comment:
Is HEEx formatter already available in this Phoenix version? I think it can solve some formatting inconsistencies

Reply:
That's with HEEx formatter applied :laughing:

Reply:
Added them manually anyway :)